libraries/core_libs/network/src/tarcap/packets_handlers/pbft_block_packet_handler.cpp

```diff
@@ -28,6 +33,19 @@ void PbftBlockPacketHandler::process(const PacketData &packet_data, const std::s
     peer->pbft_chain_size_ = peer_pbft_chain_size;
   }
 
+  // Ignore any pbft block that is missing dag anchor
```

Comment:
If this is a good PBFT proposal block, just DAG has not synced yet. We should not drop the block

Reply:
I have modified this